backend/app/controllers/api/v1/accounts/projects_controller.rb

```diff
@@ -47,6 +47,8 @@ def project_params
               :progress_impact_tracking,
               :description,
               :relevant_links,
+              :geometry,
```

Comment:
I have expected that FE will send geometry as JSON (for example `{type: "Point", coordinates: [1, 2]}`), but Rswag sends similar piece of geometry as `'{type: "Point", coordinates: [1, 2]}'` (String), therefore I am allowing this type of param at controller. Rest of the code is able to deal with this without any extra modification.